ansible/roles/schulcloud-server-tspsync/templates/api-tsp-sync-cronjob.yml.j2

```diff
@@ -0,0 +1,109 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: api-tsp-sync-cronjob
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: api-tsp-sync-cronjob
+    app.kubernetes.io/component: sync
+    app.kubernetes.io/managed-by: ansible
+    git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+    git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+  name: api-tsp-sync-cronjob
+spec:
+  concurrencyPolicy: Forbid
+  schedule: "{{ SERVER_TSP_SYNC_CRONJOB_SCHEDULE|default("9 3 * * *", true) }}"
```

Comment:
We are at the process of changing how the default  for na Ansible variable is set.
If the ansible variabele only use in one ansible role like  `SERVER_TSP_SYNC_CRONJOB_SCHEDULE` we will put the default now to `defaults/main.yml` at the ansible role as example for this one woud be the file at `ansible/roles/schulcloud-server-tspsync/defaults/main.ym`